src/components/show-edit-pod/show-edit-pod.spec.js

```diff
@@ -11,10 +11,32 @@ import {
   rootTagTest,
 } from "../../__internal__/utils/helpers/tags/tags-specs";
 import { testStyledSystemMargin } from "../../__spec_helper__/test-utils";
+import Logger from "../../__internal__/utils/logger";
+
+jest.mock("../../__internal__/utils/logger");
 
 describe("ShowEditPod", () => {
   testStyledSystemMargin((props) => <ShowEditPod {...props} />);
 
+  it("when user is trying to use a component, deprecation warning is raised in the console", () => {
```

Comment:
Just a minor tweak in wording :)
```suggestion
  it("when user uses the component, a deprecation warning is raised in the console", () => {
```